rest/route/host.go

```diff
@@ -449,6 +449,11 @@ func (ch *offboardUserHandler) Run(ctx context.Context) gimlet.Responder {
 	}
 
 	if !ch.dryRun {
+		err = ch.sc.RemoveAdminFromProjects(ch.user)
+		if err != nil {
+			return gimlet.NewJSONErrorResponse(errors.Wrap(err, "database error removing admin from projects"))
```

Comment:
_Typically_ returning an error at this point is correct. However you'll note that in this route specifically we added "terminate volumes/hosts" type errors to a catcher (it basically aggregates all non-nil errors you add to it, and returns true for HasErrors() if there are errors, and returns all the errors with `catcher.Resolve()`), and for clearLogin we only logged the error rather than returning it (grip is the package we use for logging).

This is because for this route specifically, the only error we really care about is if we weren't able to take down an unexpirable host/volume. The other errors, like this one, are more inconveniences/probably indicate a programmer error, which we'd want to know by logging it but not by returning an error early to the user.

Reply:
Sorry, when I meant you should log the error, I meant wrap it with grip.Error(). By adding it to the catcher, you're still returning it to the user, and in this case we don't really need to bother them with it.